PermanentOrg/web-app
# Per 9422 mobile banner

I have implemented a banner, that when clicked, would redirect the user to the app installed on their device, or to the store.

Steps to test if testing on a computer or laptop:
1. Open the app in the log in, onboarding or any workspace
2. Inspect the app with devtools
3. Go into responsive mode and choose either an android or ios phone
4. Refresh the page and the banner should show up
5. By clicking the banner after 1.5 seconds the user should see a prompt (this is for security reasons to please Chromium browsers) and by clicking yes they would be redirected to the store

Steps to test if on a mobile phone:
1. Open the app in the log in, onboarding or any workspace
2. Go into responsive mode and choose either an android or ios phone
3. By clicking the banner after 1.5 seconds the user should see a prompt (this is for security reasons to please Chromium browsers) and by clicking yes they would be redirected to the store, or if the app is installed, the app should open when clicking the banner without the prompt appearing.
